New known amendments from rippled 1.0.0

mDuo13 opened the pull request:
Updates the "Known Amendments" page.

Resolves DOC-1567.

mDuo13 committed:
Start updating known amendments for rippled 1.0

mDuo13 committed:
Add detailed fix1623 description

mDuo13 committed:
Add new Amendments for v1.0.0

jhaaaa commented on content/news/known-amendments.md:
```diff
@@ -225,6 +227,53 @@ Fixes a bug where validators could build consensus ledgers with different timest
+
+Adds delivered amount to metadata for CheckCash transactions cashed for a flexible amount. (Has no effect unless the [Checks](#checks) amendment is enabled.)
+
+With this amendment enabled, transaction processing adds a `DeliveredAmount` field added to the metadata of [CheckCash transactions][] for a variable amount (using the `DeliverMin` field). This change is written to the ledger data, resulting in a different ledger hash than would result from processing the transaction without this amendment. It does not affect the actual amounts delivered. Additio ...
```
Super nitpick - delete "added" - changing

> adds a `DeliveredAmount` field added to the metadata of [CheckCash transactions][]

to 

> adds a `DeliveredAmount` field to the metadata of [CheckCash transactions][]

jhaaaa reviewed (approved):
One suggested edit - otherwise looks great to me!

mDuo13 committed:
Known Amendments - fix extra 'added' typo

mDuo13 commented:
Fixed the typo and rebased on master to get [the commit that fixes the links](https://github.com/ripple/ripple-dev-portal/commit/bb09428e03512e59693cee2fe73a7fb24a2de278).
